health/service/pom.xml

```diff
@@ -16,6 +16,10 @@
       <groupId>com.cerner.beadledom</groupId>
```

Comment:
Add changes to CHANGELOG.md, add yourself to CONTRIBUTORS.md

Reply:
Added myself here: https://github.com/cerner/beadledom/pull/123/commits/bd033376b739f9d4baae3808ec2f105145d1f7c3

Do you think these changes would be classified as a defect?

Reply:
I would keep it as a defect since we shouldn't have been returning non-primary dependencies from the health resource.